### PR TITLE
Feature/detail band

### PR DESCRIPTION
**Pull request by quimpm**
Falta currar-se el front, sergi revisa-ho.

### CODE REVIEW

File: apps/main/templates/band/list.html [hunk 1]
```diff
@@ -7,10 +7,10 @@
...
         {% if band.image %}
-        <img src="{{ band.image.url }}" class="card-img-top" alt="{{ band.user.username }}">
+        <img src="{{ band.image.url }}" class="card-img-top" alt="{{ band.name }}">
         {% endif %}
         <div class="card-body">
-          <h5 class="card-title" name="list_element">{{ band.user.username }}</h5>
+          <h5 class="card-title" name="list_name">{{ band.name }}</h5>
           <p class="card-text" name="list_mail">{{ band.email }}</p>
           <p class="card-text" name="list_mobile">{{ band.mobile }}</p>
           {% comment %}<a href="{% url band_detail band.pk %}" class="btn btn-primary">View details</a>{% endcomment %}
```
Comment (sergisi) [inline apps/main/templates/band/list.html:16]: Uncomment line so lists work with detail.

File: apps/main/tests/test_detail_band.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from django.contrib.auth import get_user_model
+from apps.main.models import Band
+from django.urls import reverse
+
+User = get_user_model()
+
+
+class DetailBandTest(TestCase):
+
+    global band
```
Comment (sergisi) [inline apps/main/tests/test_detail_band.py:11]: Unused variable

Comment (quimpm) [reply apps/main/tests/test_detail_band.py:11]: Perquè m'ensenyes això? jajaja

File: apps/main/templates/event/list.html [hunk 3]
```diff
@@ -12,7 +12,7 @@ <h5 class="card-title" name="list_name">{{ event.name }} - {{ event.get_state_di
           <p class="card-text" name="list_description">{{ event.description }}</p>
           <p class="card-text" name="list_est_name">{{ event.establishment.name }}</p>
           {% comment %}<a href="{% url establishment_detail establishment.pk %}" class="btn btn-primary">View details</a>{% endcomment %}
-          <a href="#" class="btn btn-primary">View details</a>
```
Comment (sergisi) [inline apps/main/templates/event/list.html:15]: You didn't delete the comment

Comment (quimpm) [reply apps/main/templates/event/list.html:15]: Done!

**Comment by quimpm** [comment]
Perfect let's merge ma boy

**Pull request merged**